2-mandatory/5-reading-list.js

```diff
@@ -16,12 +16,55 @@ In this style of testing it is typical to write out as strings exactly what you
 without using any variables or any logic like loops, template strings or if statements.
 */
 
-const books = [];
+const books = [
+  {
+    title : "Purple",
+    author : "Chimamanda Ngozi",
+    alreadyRead: true
+  },
+
+  {
+    title : "Eat, Sleep, Prophesy",
+    author : "Tomi Arayami",
+    alreadyRead: false
+  },
+
+  {
+    title : "Woman Thou Art Loosed",
+    author : "Td Jakes",
+    alreadyRead: true
+  },
+  {
+    title : "The Journey Man",
+    author : "Jo Nesbo",
+    alreadyRead: false
+  },
+
+  {
+    title : "Half of a Yellow Sun",
+    author : "Chimamanda Ngozi",
+    alreadyRead: true
+  },
+
+];
   
 // exercise 1
 function logBooks() {
+  books.forEach(book => {
+    console.log(book.title + " " + " " + book.author)
```

Comment:
I believe the requirement is to output ``<book title>`` by ``<book author>``. You are missing the "by"